spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'UsersController', type: :request do
+  describe 'GET /index' do
+    it 'Returns http success' do
+      get '/users'
+      expect(response.status).to eq(200)
+    end
+    it 'renders the index template' do
+      get '/users'
+      expect(response).to render_template(:index)
+    end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=template%20was%20rendered.-,If%20the%20response%20body%20includes%20correct%20placeholder%20text.,-Need%20a%20big), Please include a test that ensures the response body includes the correct placeholder text.
Example : `expect(response.body).to include("Welcome to user index page")`